qancode/tasks.py

```diff
@@ -555,10 +555,11 @@ def get_data(self):
             self._expand_document_details()
         except:
             pass
-        try:
-            self._get_rid_of_test_warning_banner()
-        except:
-            pass
+        if 'https://www.encodeproject.org' not in self.driver.current_url:
```

Comment:
I couldn't figure out the root cause, but after bootstrap removal there were a few items (e.g. ENCPL000LPE, ENCSR255XZG) where the stitching of the images was totally incorrect - but only with Firefox and only on production where there isn't a test banner. Somehow, adding this check fixes the problem.